app/docs/dev/policies/meshtrace.md

```diff
@@ -42,17 +42,96 @@ For HTTP you can also manually forward the following headers:
 | MeshGatewayRoute  | ✅        | ❌  | ❌   |
 | MeshHTTPRoute     | ❌        | ❌  | ❌   |
 
-If you don't understand this table you should read [matching docs](matching#policy-matching)
+If you don't understand this table you should read [matching docs](../matching#policy-matching).
 
-## Add MeshTrace resource
+## Configuration
 
-:::::::: tabs :options="{ useUrlFragment: false }"
-::::::: tab "Zipkin"
-:::::: tabs :options="{ useUrlFragment: false }"
-::::: tab "Kubernetes"
-:::: tabs :options="{ useUrlFragment: false }"
-::: tab "Minimal example"
+### Sampling
+
+{% tip %}
+Most of the time setting only `overall` is sufficient; `random` and `client` are for advanced use cases.
```

Comment:
```suggestion
Most of the time setting only `overall` is sufficient. `random` and `client` are for advanced use cases.
```
Could we specify what those use cases are? Or are those covered in the links below?

Reply:
Yes they are covered in the Envoy links.